### PR TITLE
UI Gh-76, 111, 109, 112, 118 and 121

### PR DESCRIPTION
**Pull request by rocky341**
Ui release for the following tickets:
#76 #111 #109 #112 #118 #121

### CODE REVIEW

File: gaas-ui/src/components/cluster-namespaces/cluster-namespaces.tsx [hunk 1]
```diff
@@ -96,8 +101,11 @@ export default class ClusterNamespaces extends React.Component<{}, IState> {
                                 Refresh Table
                             </Button>
                         </Grid>
-                    </Container>
-                </Grid>
+                    </Grid>
+                    <Box pt={4}>
+                        <Copyright />
```
Comment (t92549) [inline gaas-ui/src/components/cluster-namespaces/cluster-namespaces.tsx:106]: This could maybe be on all pages?

**Review by t92549 — CHANGES_REQUESTED** [review]

File: gaas-ui/src/components/cluster-namespaces/cluster-namespaces.tsx (hunk 1 above)
Comment (rocky341) [reply gaas-ui/src/components/cluster-namespaces/cluster-namespaces.tsx:106]: Done

File: gaas-ui/.eslintrc.json [hunk 2]
```diff
@@ -20,7 +20,6 @@
     "arrow-body-style": ["error", "as-needed"],
     "quotes": [2, "double", "avoid-escape"],
     "new-cap": "error",
-    "no-invalid-this": "error",
```
[the remark this answers is not in the document]
Comment (rocky341) [reply gaas-ui/.eslintrc.json:23]: Taking a look into this. Adding the rule back breaks `src/rest/clients/rest-client.ts` but everything else seems ok. I'll get back to you on this.

[the remark this answers is not in the document]
Comment (rocky341) [reply gaas-ui/.eslintrc.json:23]: @t92549 
Rule has been added back in. RestClient has been modified slightly in order to comply to this new rule.

**Review by t92549 — APPROVED** [review]

**Pull request merged**